templates/zerver/help/include/rest-endpoints.md

```diff
@@ -10,7 +10,7 @@
 * [Remove an emoji reaction](/api/remove-reaction)
 * [Render a message](/api/render-message)
 * [Get a message's raw Markdown](/api/get-raw-message)
-* [Check messages match narrow](/api/check-narrow-matches)
+* [Check messages match narrow](/api/check_messages_match_narrow)
```

Comment:
We probably want to keep this as `-`s for consistency.

Reply:
Yes. I was going to replace all `_` by `-` in this file, but forgot before pushing.